frontend/app/actions/RegistrationUri.scala

```diff
@@ -33,10 +34,11 @@ object RegistrationUri {
     controllers.routes.Joiner.joinFriend().path -> "FRI"
   )
 
-  def parse(request: RequestHeader) = {
-    val redirectUrl: String = Config.idWebAppRegisterUrl(request.uri)
+  def parse(request: RequestHeader): String = {
```

Comment:
it would be nice if this returned a `Uri` as opposed to a string, to make its meaning more clear